cosmos-sdk-rbuilder/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.15.0-buster
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Could you add some comments to the docker file. Just a few places to show what is happening.

Reply:
Done